src/git.js

```diff
@@ -65,6 +65,29 @@ module.exports = function gitFactory(repository = '', origin = 'origin') {
                 .then( branches => branches.all);
         },
 
+        /**
+         * Check if the release branch for master exists or suggest main instead.
+         * @param {String} branchName - the branch name
+         * @returns {Promise}
+         */
+        hasReleaseBranch(branchName) {
+            return git(repository).fetch(origin)
+                .then(() => this.getLocalBranches() )
+                .then( branches => {
+                    if (!branches) {
+                        return null;
+                    }
+
+                    if (branches.indexOf(branchName) > -1) {
+                        return branchName;
+                    } else if (branchName === 'master' && branches.indexOf('main') > -1) {
```

Comment:
**thought:** It does the job. However, I wonder if there is a better way to find the base branch.